test/org/ojalgo/machine/TestBasicMachine.java

```diff
@@ -0,0 +1,63 @@
+package org.ojalgo.machine;
+
+import org.junit.jupiter.api.Test;
+import org.ojalgo.TestUtils;
+
+public class TestBasicMachine {
+    @Test
+    public void testEquals_memoryUnequal() {
+
+        final BasicMachine a = new BasicMachine(-4_756_927_793_668_063_236L, 0);
```

Comment:
These huge negative memory values are a little counter-intuitive (although not strictly a problem in the class).